mongoDB/mongo.manager.go

```diff
@@ -9,8 +9,11 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+//CNX is a mongo client
 var CNX = Connection()
 
+//Connection is a function that creates a mongo client and
+//connects it to the db and pings/checks the connection
 func Connection() *mongo.Client {
 	// Set client options
 	clientOptions := options.Client().ApplyURI("mongodb://root:example@0.0.0.0:27017")
```

Comment:
Should be an environment variable, and the port should be a different envar.
See `server.go` in Drive's api-gateway and use viper

Reply:
done